core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/kubernetesModels.scala

```diff
@@ -294,7 +294,7 @@ final case class AppName(value: String) extends AnyVal
 final case class AppResources(namespace: Namespace,
                               disk: Option[PersistentDisk],
                               services: List[KubernetesService],
-                              kubernetesServiceAccount: Option[KubernetesServiceAccount])
+                              kubernetesServiceAccountName: Option[KubernetesServiceAccountName])
```

Comment:
I'm wondering if we should make `kubernetesServiceAccountName` non-Optional in the DB and have front Leo always populate it. Back Leo would read it and create it in GKE. This is similar to how we handle namespaces.

Reply:
hmmm the reason I didn't go with that approach was in the case that the call to `kubernetesService.createServiceAccount` fails in back leo

Reply:
Yeah that's true… ok I think it's fine to only populate it in the DB after it's created in k8s